docs/user-guide/cli-using-team-profiles-hierarchy.md

```diff
@@ -0,0 +1,111 @@
+```
+
+Including properties with every command can be tedious, because there can be a lot of information required. This can lead to typos and mistakes.
+
+Another way of specifying these properties -- using configuration files -- can make things easier. A configuration file contains profiles with properties that Zowe CLI can use when you run a command.
+
+If configuration files were used in the example above, the user would have needed to issue only the command: 
+
+`zowe zos-files list data-set "SYS1.PARMLIB*"`
+
+## Learning the terminology
+
+Zowe version 2.0 introduces the use of team profiles in configuration files. Both *user* and *team* profiles are stored in configuration files, and these configuration files can either be *project* configuration files or *global* configuration files. It is helpful to understand how these differ.
+
+- A **user configuration file** stores *user profiles* and is used for one person who needs their own unique properties to run commands.
+
+- A **team configuration file** stores *team profiles* and is used by a group of people who need the same properties to run commands.
+
+- A **global configuration file** resides in the `ZOWE_CLI_HOME` directory (YourUserHomeDirectory/.zowe, by default). It contains global *user profiles* and global *team profiles*.
+
+- A **project configuration file** resides in a directory of your choice. It contains project *user profiles* and project *team profiles*.
+
+All configuration files are saved in `.json` format.
+
+## How configuration files and profiles work together
+
+There may be instances where a user has all four types of files in their system, and all four configurations are referred to simultaneously by Zowe CLI for a particular command. **[is "simultaneously" correct?]**
+
+This can mean working with files that have contradictory configurations. One file can specify that a certain profile property has a value of `ABC`, while another file uses `XYZ` as a value.
+
+When the same properties have different values across multiple configuration files, Zowe CLI follows a two-step check to determine which configurations apply:
+
+1. Does the configuration file have a more *specialized* or more *generalized* use?
+2. Is the configuration file *more specific* or *less specific*.
+
+Zowe CLI considers a user configuration file to have a more specific use than a team configuration file, and a project configuration file to be more specialized than a global configuration file, which is more generalized.
+
+When checking all possible configuration file types, Zowe CLI categorizes files in the manner below:
+
+![Zowe CLI Config File Table](../images/common/cli-config-file-table.jpg)
+
+This order is applied no matter the directory in which you issue a Zowe CLI command. As a user, it can be easy to trace this logic when configuration files are all either in your home directory (i.e., specialized) or your project directory (i.e., generalized).
```

Comment:
I suggest that we change "your home directory"
to "your ZOWE_CLI_HOME directory.

Reply:
Done -- thank you!